### PR TITLE
GAC costing documentation into template dedicated file

### PR DESCRIPTION
**Pull request by hunterbarber**
## Fixes/Resolves:
GAC costing documentation for #1360 
Mostly lateral changes from unit model documentation to costing model documentation files with minor wording changes

## Summary/Motivation:
Cost documentation for all unit models

## Changes proposed in this PR:
- Move costing documentation from GAC unit model doc to new cost model documentation generated from the template

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/costing/gac.rst [hunk 1]
```diff
@@ -4,12 +4,40 @@ Granular Activated Carbon Costing Method
...
+   "Number of contactors operating in parallel", ":math:`N_{op}`", "num_contactors_op", "1", ":math:`\text{dimensionless}`"
+   "Number of redundant contactors in parallel", ":math:`N_{red}`", "num_contactors_redundant", "1", ":math:`\text{dimensionless}`"
+   "Fraction of spent GAC adsorbent to be regenerated and reused", ":math:`f_{regen}`", "regen_frac", "0.70", ":math:`\text{dimensionless}`"
+   "Reference maximum value of GAC initial charge mass where economy of scale no longer discounts the unit price (U.S. EPA, 2021)", ":math:`M_{GAC}^{ref}`", "bed_mass_max_ref", "18143.7", ":math:`kg`"
+   "Contactor polynomial cost coefficients", ":math:`x`", "contactor_cost_coeff", "tabulated", ":math:`\text{dimensionless}`"
+   "Adsorbent exponential cost coefficients", ":math:`y`", "adsorbent_unit_cost_coeff", "tabulated", ":math:`\text{dimensionless}`"
+   "Other process costs power law coefficients", ":math:`z`", "other_cost_param", "tabulated", ":math:`\text{dimensionless}`"
+   "Unit cost to regenerate spent GAC adsorbent", ":math:`C_{unit,regen}`", "regen_unit_cost", "4.28352", ":math:`USD\_2020/kg`"
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/gac.rst]:
```suggestion
   "Unit cost to regenerate spent GAC adsorbent", ":math:`C_{unit,regen}`", "regen_unit_cost", "4.28352", ":math:`\text{USD}_{2020}\text{/kg}`"
```

**Review by MarcusHolly — APPROVED** [review]
LGTM, my only suggestion would be to replace instances of ``USD\_2020`` with what I added in the comment below.

File: docs/technical_reference/costing/gac.rst (hunk 1 above)
Comment (hunterbarber) [reply docs/technical_reference/costing/gac.rst]: Yea, I didn't know what the best formatting would be there. I also considered like a $_{YYYY} but nothing felt too great for including the year in the default parameter units. Is this what you used across the flowsheets you are rolling out?

Comment (MarcusHolly) [reply docs/technical_reference/costing/gac.rst]: Yes, this is what I've been using in my costing documentation (#1373)

**Review by savannahsakhai — APPROVED** [review]
LGTM.

**Comment by lbianchi-lbl** [comment]
CI failures are unrelated and most likely due to #1158. I've restarted the failed job which should result in this PR getting automerged once it passes.

**Pull request merged**